### DISCUSSION TITLE
How to load downloaded level?

### DISCUSSION THREAD

**Discussion by mariomadproductions** [city41/smaghetti#170] [Q&A]
I know I can download my levels as save files, but how do I load these again? Or do I need to backup the webapp's local storage somehow to be able to do that?

**Comment by city41** [comment]
Unfortunately that is not quite possible yet. 

You can download your level as a `.level` file by clicking on the beaker in the lower right to bring up experiments then click "Download level as binary".

The downloaded .level file is legit. It will load in Solar Magic Advance and other tools no problem. But Smaghetti's ability to read a .level file (in the same experiment, "Load binary level") is a work in progress. It's a huuuuuuuuge task too, I've been chipping away at it for a long time with no end in sight, sadly.

**Reply by mariomadproductions** [reply]
I see, thanks. Is there a way to just save the current state of the webapp somehow? I had a look in the "Application" tab of chrome devtools, but I couldn't find an obvious thing to save.

**Reply by city41** [reply]
Not currently as most people are ok with saving their level to the server.
